src/sakshi/Assignment19.java

```diff
@@ -0,0 +1,42 @@
+/*1. Print the frequecy of given character from given String.
+input : technocredits
+        t
+		
+output : frequency of t in technocredits is 2.
+
+2. Print character of given String in reverse order.
+input : techno
+output : 
+o
+n
+h
+e
+t*/
+
+package sakshi;
+
+public class Assignment19 {
+
+	void getFrequency(String name) {
+		int count=0;
+		for(int index=0; index<name.length(); index++) {
+			if(name.charAt(index) == 'e') {
```

Comment:
use generic process, your code will only work for character 'e', you have to pass any character as a parameter and your code will print frequency of it